### PR TITLE
Task-36530: Add users role for /platform/externals group members

### PR DESCRIPTION
**Pull request by azayati**

### CODE REVIEW

**Review by plamarque — DISMISSED** [review]
unit tests

File: exo.core.component.security.core/src/main/java/org/exoplatform/services/security/impl/DefaultRolesExtractorImpl.java [hunk 1]
```diff
@@ -35,55 +35,45 @@
...
+        roles.add(splittedGroupName[splittedGroupName.length - 1]);
       }
-      return roles;
-   }
-
+      else {
+        roles.add(splittedGroupName[0]);
+      }
+    }
+    if (roles.contains("externals") && !roles.contains("users")) {
```
Comment (boubaker) [inline exo.core.component.security.core/src/main/java/org/exoplatform/services/security/impl/DefaultRolesExtractorImpl.java:74]: What is this group (`externals`) for ?
If it's a specific need, we should make it configurable by `exo.properties` and not embed it.
This is a huge modification that will change the default behavior of the platform for a specific need.
Please consider adding custom addon to implement custom behaviors.

**Review by boubaker — DISMISSED** [review]

**Review by rdenarie — APPROVED** [review]

File: exo.core.component.security.core/src/main/java/org/exoplatform/services/security/impl/DefaultRolesExtractorImpl.java (hunk 1 above)
Comment (azayati) [reply exo.core.component.security.core/src/main/java/org/exoplatform/services/security/impl/DefaultRolesExtractorImpl.java:74]: already discussed

**Review by ahmadmido — APPROVED** [review]
LGTM

**Pull request merged**